### DISCUSSION TITLE
Clarification of project contributor doubts

### DISCUSSION THREAD

**Discussion by 0904-mansi** [kubeedge/kubeedge#3793] [Q&A]
Hey if we want to contribute to this project then is there any task to complete before contributing to the project? and any prerequisite that 2-3 PRs should be merged before the GSoC period begins?

**Comment by vincentgoat** [comment]
Hi @0904-mansi, please check this https://github.com/kubeedge/kubeedge/blob/master/CONTRIBUTING.md
